gensim/models/lda_worker.py

```diff
@@ -4,13 +4,13 @@
 # Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
 
-"""
-Worker ("slave") process used in computing distributed LDA. Run this script \
-on every node in your cluster. If you wish, you may even run it multiple times \
-on a single machine, to make better use of multiple cores (just beware that \
-memory footprint increases accordingly).
+"""Worker ("slave") process used in computing distributed LDA.
```

Comment:
First of all, please fix PEP8 problems (almost lead spaces), look at travis log https://travis-ci.org/RaRe-Technologies/gensim/jobs/342495787#L511

Reply:
Ok. Also, should I add a section for module level attributes such as *HUGE_TIMEOUT* ,*MAX_JOBS_QUEUE*,etc  in lda_dispatcher.py ?